apps/api/src/app/groups/groups.controller.ts

```diff
@@ -365,6 +377,52 @@ export class GroupsController {
         throw new NotImplementedException()
     }
 
+    @Post("/member/:member")
```

Comment:
```suggestion
    @Post("/members/:member")
```

What about using `members` instead of `member` here for consistency?

Reply:
My original thought was to not confuse users with the plural `members` since this only support adding 1 `member`, but now looking at `/members/:member` is also used for single user apis too.

The change makes sense, will update it now.